docs/core/evaluation.rst

```diff
@@ -158,4 +158,51 @@ you may do so by adding the ``e2e=true`` query parameter:
 
   $ curl --data-binary @eval_stories.md "localhost:5005/evaluate?e2e=true" | python -m json.tool
 
+Test files for possible mistakes
```

Comment:
@MetcalfeTom I reckon this should actually go under `Debugging` instead of `Evaluating and Testing`, no?

Reply:
Should I change it to this other file then?

Reply:
Looks like tom agrees, so yes that would be great!

Reply:
@gabibguedes would also be good if `utters` could be changed to `utterances` (and `utter` to `utterance`) overall in your code too

Reply:
Alright then.